src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -271,11 +281,11 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             _renderEngine->SetSoftwareRendering(_settings.SoftwareRendering());
             _renderEngine->SetIntenseIsBold(_settings.IntenseIsBold());
 
-            _updateAntiAliasingMode(_renderEngine.get());
+            _updateAntiAliasingMode();
 
             // GH#5098: Inform the engine of the opacity of the default text background.
             // GH#11315: Always do this, even if they don't have acrylic on.
-            _renderEngine->SetDefaultTextBackgroundOpacity(::base::saturated_cast<float>(_settings.Opacity()));
+            _renderEngine->SetDefaultTextBackgroundOpacity(static_cast<float>(_settings.Opacity()) * static_cast<int>(_settings.BackgroundImage().empty()));

```

Comment:
Yeah this is a terrible hack. I think I'll open another PR first where I rename `SetDefaultTextBackgroundOpacity` into something like "UseAlphaForBackground(bool)" or something. This change is needed, because this only works in `DxEngine` due to random chance and depends on the fact that DxEngine defaults to ClearType, which AtlasEngine can't yet.

Reply:
You know this is actually wrong. I just remembered the real reason:
While the D2D-based DxEngine only draws what needs to be drawn, the D3D-shader-based AtlasEngine needs to do alpha blending with the VT foreground and background colors for all pixels. If you don't enable alpha explicitly in WT (which breaks conhost support and thus isn't neabled by default) then AtlasEngine will draw everything opaque.

Reply:
Oh right this issue is still unresolved. I should open a PR for this tomorrow... But it will conflict with Mike's... 🤔
Well... can't help it!